recipes/klongpy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "klongpy" %}
+{% set version = "0.3.61" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/klongpy-{{ version }}.tar.gz
+  sha256: be4981b8621fc5eb3d74948404e8272830e4da3eb52c800b632fb52ad0e0afdc
+
+build:
+  skip: true  # [py<38]
```

Comment:
Can you add a comment that this one has scripts and cannot be noarch? Or, better yet, use the "semi-arch" new syntax and make it a win and nix noarch package?

Reply:
@ocepaf can you remind me where the documentation of this semi-arch syntax might be?

Reply:
You are only skipping Python, right? So why not vanilla noarch?

Reply:
PS: here is an example of a Windows only noarch: https://github.com/jaimergp/win_inet_pton-feedstock/blob/209342dfb957b8aafc8094f703849bc3becad7a2/recipe/meta.yaml

Reply:
@ocefpaf Still a little confused, I'm thinking this has scripts and therefore cannot be noarch, but would adding `__win` to dependency make this work as noarch even though it has scripts? Happy to look at documentations on Conda about this if it exists

Reply:
Ah. That was the reason. I forget. To make it noarch for python for 1 package for each platform is a bit more involved. Let's merge this and we can work on it later in the feedstock.

Reply:
Appreciate it @ocepaf I'll ping you in the repo once it materializes